src/drivers/amqp/Queue.php

```diff
@@ -109,21 +126,58 @@ public function status($id)
      */
     protected function open()
     {
-        if ($this->channel) return;
-        $this->connection = new AMQPStreamConnection($this->host, $this->port, $this->user, $this->password);
-        $this->channel = $this->connection->channel();
-        $this->channel->queue_declare($this->queueName, false, true, false, false);
-        $this->channel->exchange_declare($this->exchangeName, 'direct', false, true, false);
-        $this->channel->queue_bind($this->queueName, $this->exchangeName);
+        if ($this->context) return;
```

Comment:
Add `{` and `}` please.

Reply:
sure, (that's how it was, I kept original styling).